Resources/Private/Fusion/Document/Base.fusion

```diff
@@ -11,13 +11,13 @@ prototype(Neos.Demo:Document.Base) < prototype(Neos.Fusion:Component) {
             stylesheets {
                 site = Neos.Demo:Document.Fragment.StyleSheets
             }
+            javascripts {
+                site = Neos.Demo:Document.Fragment.JavaScripts
+            }
             metadata = Neos.Demo:Document.Fragment.MetaData
         }
 
         bodyTag.attributes.class = ${props.bodyClass}
         body = ${props.body}
-
```

Comment:
Why are they moved to the head?

Reply:
Because it is better to have them in the head with the attribute `defer`: Start loading and wait till the document is ready.